lib/hammer_cli_katello/repository.rb

```diff
@@ -75,6 +76,7 @@ class InfoCommand < HammerCLIKatello::InfoCommand
       def extend_data(data)
```

Comment:
I would try to separate some of this logic into a method, like `setup_sync_state` does.